colorizer_data/converter.py

```diff
@@ -334,7 +334,7 @@ def _should_regenerate_frames(
                 logging.info(f"Frame {frame} is missing. Regenerating all frames.")
                 return True
     # Get object count and regenerate frames if it has changed
-    num_objects = data[config.object_id_column].nunique()
+    num_objects = len(data[config.times_column])
```

Comment:
Using `nunique` instead of the length of the data was the problem. I also had this pull from the times column since it's comparing against times data.